phpstan.neon.dist

```diff
@@ -0,0 +1,17 @@
+includes:
+    - ./vendor/nunomaduro/larastan/extension.neon
+    - ./phpstan-baseline.neon
+
+parameters:
+    tmpDir: .cache/phpstan
+
+    paths:
+        - src
+
+    level: 2
+
+    editorUrl: 'phpstorm://open?file=%%file%%&line=%%line%%'
+
+    ignoreErrors:
+        - '#Unsafe usage of new static#'
```

Comment:
These issues were the remaining errors from level 1, I wasn't sure how to fix them. So I chose to ignore them.